web/gds-user-ui/src/components/Section/SearchDirectory.tsx

```diff
@@ -83,15 +98,32 @@ const SearchDirectory: React.FC<TSearchDirectory> = ({
               <form onSubmit={(e) => handleSubmit(e, search)}>
                 <FormControl color={'gray.500'}>
                   <HStack>
-                    <Input
-                      size="md"
-                      width={'100%'}
-                      type="search"
-                      isRequired
-                      placeholder={t`Common name or VASP ID`}
-                      name="search"
-                      onChange={(event: any) => setSearch(event.currentTarget.value)}
-                    />
+                    <AutoComplete
+                      rollNavigation
+                      // no found message
+                      noOptionsMessage="No VASP found">
+                      <AutoCompleteInput
+                        variant="filled"
+                        placeholder="Common name or VASP ID"
+                        autoFocus
+                      />
+                      <AutoCompleteList>
+                        {Object.keys(options)?.map((oid: any) => (
+                          <AutoCompleteItem
```

Comment:
I noticed that more than one value had the hover background when I attempted a search and in the acceptance criteria video. 

Here's a video of what I experienced: https://www.awesomescreenshot.com/video/20164689?key=29b03758f4a12fb65f5bedf10e4f6b1f